modules/oci-image/01_mod.mk

```diff
@@ -45,6 +45,8 @@ ifeq ($(oci_$1_base_image_flavor),static)
     oci_$1_base_image := $(base_image_static)
 else ifeq ($(oci_$1_base_image_flavor),csi-static)
     oci_$1_base_image := $(base_image_csi-static)
+else ifeq ($(oci_$1_base_image_flavor),custom)
+	$(call fatal_if_undefined,oci_$1_base_image)
```

Comment:
@ThatsMrTalbot
I think this should have been `$$(call fatal_if_undefined,oci_$1_base_image)`
Right now, this breaks eg. approver-policy with the following error:
```console
make/_shared/oci-image/01_mod.mk:75: *** oci_manager_base_image is not set.  Stop.
```